ts/webui/src/components/experiment/trialdetail/TrialsDetail.tsx

```diff
@@ -48,8 +48,7 @@ class TrialsDetail extends React.Component<{}, TrialDetailState> {
     render(): React.ReactNode {
         const { whichChart } = this.state;
         const source = TRIALS.toArray();
-        const trialIds = TRIALS.toArray().map(trial => trial.id);
-
+        const trialIds = TRIALS.succeededTrials().map(trial => trial.id);
```

Comment:
Rename this to `succeededTrialIds` or whatever.

Reply:
yes!